### ISSUE TITLE
Remove checking for local cli rpm if installing with internet connectivity

### ISSUE THREAD

**Issue by myersCody** [quipucords/quipucords-installer#40]
There is an issue with the logic for installing the cli rpm. Recently we switched to using yum for most of our rpm installs; however, there are still a couple of rpm install commands in the installer. One advantage to using yum over rpm is that yum will automatically try to install the sub dependencies to the rpm for you; whereas, the `rpm install` command will not. I believe this subtle difference is causing an issue when we try to install the client online, but a local client rpm is located in the packages directory. 

For example,
The current logic for the `cli install` ansible role task for both `online` & `offline` is:
```
check to see if local rpm exists
if it exists:
     rpm install
else:
    grabs it from internet & does a yum install
```
This means that we need to have the prerequisites installed not only for offline installs, but anytime that we are doing an `rpm install` instead of a `yum install` in the ansible tasks.

**TLDR;**
If a user is installing online always grab it from the internet and implement a yum install even if a local rpm in the packages directory exists.

**Comment by Dostonbek1** [comment]
This issue got resolved as a result of breaking up Ansible roles into smaller more specific subroles. Now since we have separate roles for installing qpc online and offline, and they run on install_offline condition, this issue is no longer there. Refer to https://github.com/quipucords/quipucords-installer/pull/57
